[Edit] Python Strings `.count()` Method (#1863)
Codecademy/docs: content/python/concepts/strings/terms/count/count.md

  - 'paths/analyze-data-with-python'
---

> The **.count()** finds the number of times the specified substring occurs within a given string.

Reviewer: ```suggestion
The **`.count()`** method finds the number of times the specified substring occurs within a given string.
```

> The **`.count()`** method finds the number of times the specified substring occurs within a given string.

Reviewer 2: Should there be a mention of case sensitivity?

The **`.count()`** method finds the number of times the specified substring occurs within a given string.

## Syntax

> ```py

Reviewer 2: This block of Python is never closed!

Author: @HishamT Are you looking at this on the mobile app? It shows everything as combined but all this was removed.

Reviewer 2: No, I am on my computer. I can't see the closing, but I will take your word for it. :grin:

```pseudo
string.count(substring, start=..., end=...)
```

> - `substring` is the substring to search for

Reviewer 2: ```suggestion
- `substring` is the substring to search for.
```

> - `start` and `end` are optional and respectively specifies a starting and end position for the search

Reviewer 2: ```suggestion
`start` and `end` are optional and specify the search's starting and end positions.
```

## Example 1
- `substring` is the substring to search for
- `start` and `end` are optional and respectively specifies a starting and end position for the search

Use `.count()` to count the number of occurrences of the string `"cat"` in the `pets` list:
> **Note:** When not specified, `start` defaults to `0` and `end` defaults to `len(string)` which means the search will be performed on the whole string.

> ```codebyte/python

Reviewer 2: I can't find the closing ``` for this code block.

Author: @HishamT Look at my previous comment.
